recipes/blend2d/all/conanfile.py

```diff
@@ -56,6 +57,9 @@ def validate(self):
             # https://github.com/blend2d/blend2d/commit/63db360c7eb2c1c3ca9cd92a867dbb23dc95ca7d
             check_min_vs(self, 192)
 
+    def build_requirements(self):
+        self.tool_requires("cmake/[>=3.18 <4]")
```

Comment:
```suggestion
        if Version(self.version) >= "0.11.1":
            self.tool_requires("cmake/[>=3.18 <4]")
```

I would not bother with older versions as they only need CMake 3.8